MapboxDirections/MBDirections.swift

```diff
@@ -165,9 +165,9 @@ open class Directions: NSObject {
      */
     @objc(calculateDirectionsWithOptions:completionHandler:)
     open func calculate(_ options: RouteOptions, completionHandler: @escaping CompletionHandler) -> URLSessionDataTask {
-        let url = urlForCalculating(options)
-        let task = dataTaskWithURL(url, completionHandler: { (json) in
-            let response = options.response(json)
+        let url = self.url(forCalculating: options)
```

Comment:
The `self.` is needed here to avoid a warning about using `url` in its own definition.